### PR TITLE
Add clarification around statusList and revocation support for Domain Linkage Credentials

### LINKED ISSUE #12: Update readability of Section 5.5
Perhaps, add new subsection
5.5.1 Structure of Authentication Response

### LINKED ISSUE #24: Wallets should not be required to check revocation status for linked domain credentials
See title.

Easier for the issuers to remove them from .well-known endpoints.

### PR DESCRIPTION
**Pull request by nklomp**
closes #24

### CODE REVIEW

**Comment by nklomp** [comment]
Please be aware I used the wrong issue nr in my commit message. Double check whether #12 gets closed by accident after merging this PR (should be #24)

File: spec/spec.md
Comment (Sakurann) [inline spec/spec.md:335]: LGTM. I wonder if we could make the last sentence a little clearer?
```suggestion
Validation of Domain Linkage Credentials by the wallet MUST follow the steps given in the [[ref: Well Known DID]] specification. Additional checks, e.g. of revocation, are not required by this profile. Since Domain Linkage Credentials are being managed by the Verifier/RP, updating the DID Configuration Resource directly, the usage of a credentialStatus property for revocation in a Domain Linkage Credential typically is of little use. To check validity of the Domain Linkage Credential, expiration property can be taken into account.
```

Comment (nklomp) [reply spec/spec.md:335]: Can suggests optionality, whilst it is required in the well-known did spec.

Maybe just have 'To check validity of the Domain Linkage Credentials, expiration needs to be taken into account`?

Comment (Sakurann) [reply spec/spec.md:335]: `exp` is optional in a Domain Linkage Assertion, right?

> The JWT object SHOULD contain the exp property, and if present, the value MUST be an epoch time that is greater than the current epoch time. If this property is not present, the assertion's future expiry is based on its continued presence in the resource (or its replacement with updated assertion that contains an expiry value).

https://identity.foundation/specs/did-configuration/#validation-of-domain-linkage-assertions

Comment (Sakurann) [reply spec/spec.md:335]: probably also align the language in the profile to the language in the .well-known and change `Domain Linkage Credential` to `Domain Linkage Assertion`?

Comment (nklomp) [reply spec/spec.md:335]: @Sakurann That looks like an outdated or initial version of the spec. The current spec mentions that expiration date must be present and calls them Domain Linkage Credentials: https://identity.foundation/.well-known/resources/did-configuration/#domain-linkage-credential

Comment (Sakurann) [reply spec/spec.md:335]: Understood. I updated the suggested text above. happy to approve the PR if you incorporate those changes.

Comment (nklomp) [reply spec/spec.md]: Updated with your suggestion. PR should be ready for approval

**Comment by nklomp** [comment]
@Sakurann PR should be ready for approval

**Review by Sakurann — APPROVED** [review]

**Pull request merged**
